src/main/kotlin/org/opensearch/commons/alerting/AlertingPluginInterface.kt

```diff
@@ -50,6 +51,31 @@ object AlertingPluginInterface {
             }
         )
     }
+    /**
```

Comment:
let's not overload index monitor method. Let's change the original method's signature

Reply:
Yeah sure. Tnx